plugins/orchestrator-common/src/auto-generated/api/models/schema.ts

```diff
@@ -142,7 +142,8 @@ export interface components {
       | 'Error'
       | 'Completed'
       | 'Aborted'
-      | 'Suspended';
+      | 'Suspended'
+      | 'Pending';
```

Comment:
@caponetto 
this schema is auto generated from ` plugins/orchestrator-common/src/openapi/openapi.yaml` spec.
Please change the spec, and then run `yarn --cwd plugins/orchestrator-common openapi`

The files that will be automatically updated are:
`plugins/orchestrator-common/src/auto-generated/api/definition.ts`,
`plugins/orchestrator-common/src/auto-generated/api/models/schema.ts`

Reply:
Oops, on it.